### PR TITLE
Add an option to add also event sources, small translation cleanups

### LINKED ISSUE #800: Events Subscription should have multiple filters with Event Source and Event Type being editable
**Description**

Events Subscription should have multiple filters with Event Source and Event Type being editable

**Expected result**

Events Subscription should have multiple filters with Event Source and Event Type being editable. We should have an input field in the advanced view where EventSource can me modified.

**Actual result**

We have just Event Type being editable

**Steps to reproduce**

**Troubleshooting**

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

Resolves #800

### CODE REVIEW

**Review by Wawrzyn321 — DISMISSED** [review]

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -1617,19 +1617,26 @@ stateful-sets:
   name_singular: Stateful Set
   replicas: Replicas
   title: Stateful Sets
-subscription:
+subscriptions:
   create:
     labels:
+      add-filter: Add Filter
```
Comment (grego952) [inline core/src/i18n/en.yaml:1623]:
```suggestion
      add-filter: Add filter
```

**Review by Wawrzyn321 — APPROVED** [review]

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -1617,19 +1617,26 @@ stateful-sets:
   name_singular: Stateful Set
   replicas: Replicas
   title: Stateful Sets
-subscription:
+subscriptions:
   create:
     labels:
+      add-filter: Add filter
```
Comment (majakurcius) [inline core/src/i18n/en.yaml:1623]: Can you show me where this is displayed? The reason I'm asking is it's a sentence, but doesn't end with a comma.

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -1617,19 +1617,26 @@ stateful-sets:
...
+      filter: Filter {{name}}
+      filters: Filters
       service-name: Service
       sink: Sink
+    messages:
+      one-filter-required: At least one filter is required.
     placeholders:
       application-name: Choose an Application name
       event-name: Enter the Event name, for example, order.cancelled
+      event-source: Enter the Event source
```
Comment (majakurcius) [inline core/src/i18n/en.yaml:1639]:
```suggestion
      event-source: Enter the event source
```

Events are capitalized only when the meaning behind it is the [Kubernetes resource](https://v1-19.docs.kubernetes.io/docs/tasks/debug-application-cluster/events-stackdriver/), not our regular Eventing events. See the [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalized-terms) for reference. 

Please correct the whole file (and any other if needed).

File: core/src/i18n/en.yaml (hunk 2 above)
Comment (dariadomagala) [reply core/src/i18n/en.yaml:1623]: Screen Shot 2022-01-14 at 11 56 21
It's in a button

Comment (majakurcius) [reply core/src/i18n/en.yaml:1623]: Ah, then all good, thank you :)

**Review by Wawrzyn321 — DISMISSED** [review]

File: tests/tests/in-cluster-eventing.spec.js [hunk 4]
```diff
@@ -43,7 +43,7 @@ context('In-cluster eventing', () => {
 
     cy.getIframeBody()
       .find(
-        '[placeholder="Enter Event type, for example, sap.kyma.custom.test-app.order.cancelled.v1"]',
+        '[placeholder="Enter event type, for example, sap.kyma.custom.test-app.order.cancelled.v1"]',
```
Comment (majakurcius) [inline tests/tests/in-cluster-eventing.spec.js:46]: I can't add a suggestion here, but don't you need `the` here? 
You have it [here](https://github.com/kyma-project/busola/blob/cae15a57d5f5a51b5f55d730d9e3419356e46555/core/src/i18n/en.yaml#L1640).

Comment (dariadomagala) [reply tests/tests/in-cluster-eventing.spec.js]: Of course I do, I fixed it one place and forgot about this one, good catch! 🙂

Comment (majakurcius) [reply tests/tests/in-cluster-eventing.spec.js]: Perfect, thank you! 😊

**Review by majakurcius — APPROVED** [review]

**Comment by Wawrzyn321** [comment]
Luigi failed in replicaset test
/test pre-busola-integration-test-k3s

**Review by Wawrzyn321 — APPROVED** [review]

**Pull request merged**